packages/obojobo-document-engine/ObojoboDraft/Chunks/ActionButton/converter.js

```diff
@@ -0,0 +1,51 @@
+const slateToObo = node => {
+	const json = {}
+	json.id = node.key
+	json.type = node.type
+	json.content = {}
+	const nodeContent = node.data.get('content')
+	json.content.label = nodeContent.label || ''
+	json.content.triggers = [
+		{
+			type: 'onClick',
+			actions: nodeContent.actions.map(action => {
+				return {
+					type: action.type,
+					value: action.value !== '' ? JSON.parse(action.value) : {}
+				}
+			})
+		}
+	]
+
+	json.children = []
+	return json
+}
+
+const oboToSlate = node => {
+	const json = {}
+	json.object = 'block'
+	json.key = node.id
+	json.type = node.type
+
+	json.data = { content: {} }
+	json.data.content.label = node.content.label
+	if (!json.data.content.label && node.content.textGroup) {
+		node.content.textGroup.forEach(line => {
+			json.data.content.label = line.text.value
```

Comment:
This appears to be going through every line but throwing away all but the last one?